bahmnicore-api/pom.xml

```diff
@@ -184,6 +184,11 @@
             <groupId>org.jcodec</groupId>
             <artifactId>jcodec-javase</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.openmrs.api</groupId>
+            <artifactId>openmrs-api</artifactId>
+            <version>2.1.1-SNAPSHOT</version>
+        </dependency>
```

Comment:
Bahmni Core API already depends on OpenMRS API (every OpenMRS module does).
See [here](https://github.com/Bahmni/bahmni-core/blob/9330997164d2f3c9e037d78762af2c87a43dae1a/bahmnicore-api/pom.xml#L31-L35) and [here](https://github.com/Bahmni/bahmni-core/blob/9330997164d2f3c9e037d78762af2c87a43dae1a/pom.xml#L27).